internal/olm/operator/uninstall.go

```diff
@@ -53,6 +54,11 @@ func NewUninstall(cfg *Configuration) *Uninstall {
 	}
 }
 
+func (u *Uninstall) BindFlags(fs *pflag.FlagSet) {
+	fs.BoolVar(&u.DeleteCRDs, "delete-crds", true, "If set to false, owned CRDs and CRs will not be deleted")
+	fs.BoolVar(&u.DeleteAll, "delete-all", true, "If set to true, it will enable all the delete flags")
```

Comment:
What about `DeleteOperatorGroups`?

Reply:
Also needs some validation so `--delete-crds=false --delete-all=true` cannot be set.

Reply:
Good catch. Added some validation. 

`./build/operator-sdk cleanup example-operator --delete-all=true --delete-crds=false --delete-operator-groups=true`
FATA[0000] Flag validation error: If --delete-all flag is enabled, any other delete flag cannot be set to false